### DISCUSSION TITLE
Unable to run on radeon RX6600

### DISCUSSION THREAD

**Discussion by warlockhb** [bnpr/Malt#471] [Q&A] [answered]
<img width="1920" alt="KakaoTalk_20230116_090943550" src="https://user-images.githubusercontent.com/71178397/212575299-56d0bca4-c41d-4383-9512-d74bae88d0dd.png">

<img width="310" alt="KakaoTalk_20230116_091036962" src="https://user-images.githubusercontent.com/71178397/212575307-aafaef4e-8a25-4cf8-90f3-062c059b5260.png">

<img width="614" alt="KakaoTalk_20230116_091254277" src="https://user-images.githubusercontent.com/71178397/212575309-8726e94a-5e4f-4e77-8ffa-8c7ccb196d7e.png">

OpenGL version also looks normal. I don't know why it can't run

what can i try?

**Comment by dibli-goost** [comment, marked as answer]
There is a known issue regarding AMD hardware which prevents Malt from working, see: #451 

You'll need to downgrade your graphics drivers in order for Malt to work properly right now. I've been stuck on 30.0.21023.1015 (Adrenalin edition 22.6.1) for a while.

**Reply by warlockhb** [reply]
After reading this comment, issue was resolved 👍👍

The way was to downgrade the AMD version

**Comment by pragma37** [comment]
Yes, this looks the same as #451.

@warlockhb You can check if this version works for you to confirm if that's the case.

**Reply by pragma37** [reply]
BTW, the Q&A section is for "how to" questions, for bug reports use the [Issues](https://github.com/bnpr/Malt/issues) section.

**Reply by warlockhb** [reply]
Thanks also to the developer who confirmed it 😄😄
